gnome-screenlock-password-opensuse132-20160620.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "type": "match",
+      "ypos": 237,
+      "height": 263,
+      "width": 458,
+      "xpos": 290
+    }
+  ],
+  "tags": [
+    "gnome-screenlock-password"

```

Comment:
mitiao, please add the corresponding tags as in screenlock-gnome-opensuse132-20160619.json, i.e ENV-DESKTOP-gnome, ENV-DISTRI-opensuse, ENV-VERSION-13.2 as the needle already has the specific requirements in the name.